src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/Switch.cs

```diff
@@ -50,11 +50,11 @@ private object InitializedLock
         /// <para>Initializes a new instance of the <see cref='System.Diagnostics.Switch'/>
         /// class.</para>
         /// </devdoc>
-        protected Switch(string displayName, string description) : this(displayName, description, "0")
+        protected Switch(string displayName, string? description) : this(displayName, description, "0")
         {
         }
 
-        protected Switch(string displayName, string description, string defaultSwitchValue)
+        protected Switch(string displayName, string? description, string defaultSwitchValue)
```

Comment:
`displayName` is allowed to be `null`, below it just changes it to `string.Empty`.

Reply:
Yes, but the comment below make me think that intention is non null:

> // displayName is used as a hashtable key, so it can never
             // be null.

Reply:
Fair enough, I agree the intention is that this string isn’t null.